packages/core/nuxt-theme-module/theme/components/AppHeader.vue

```diff
@@ -46,9 +46,9 @@ export default {
   setup(props, { root }) {
     const { toggleCartSidebar, toggleWishlistSidebar, toggleLoginModal } = useUiState();
     const { changeSearchTerm, getFacetsFromURL } = useUiHelpers();
-    const { isAuthenticated, load } = useUser();
-    const { cart, loadCart } = useCart();
-    const { loadWishlist } = useWishlist();
+    const { isAuthenticated, load: loadUser } = useUser();
```

Comment:
how about something like
```js
const $user = useUser()
const $cart = useCart()
// later
$user.load()
$cart.load()
```

Subjectively it's cleaner and can be a convention but I'm curious about your opinion

Reply:
I like this part. What about you @andrzejewsky ?